src/licensedcode/data/rules/tsl_2.yml

```diff
@@ -0,0 +1,3 @@
+license_expression: tsl-2020
+is_license_notice: yes
+relevance: 100
```

Comment:
```suggestion
relevance: 99
referenced_filenames:
    - LICENSE-TIMESCALE
notes: this could be also the tsl-2018 therefore we do not have a relevance of 100 
```